controlplane/internal/controller/openshiftassistedcontrolplane_controller_test.go

```
@@ -1064,6 +1063,519 @@ var _ = Describe("Scale operations and machine updates", func() {
+
+			for i := 0; i < 3; i++ {
+				_, err := controllerReconciler.Reconcile(ctx, reconcile.Request{NamespacedName: typeNamespacedName})
+				Expect(err).NotTo(HaveOccurred())
+			}
+
+			machineList := &clusterv1.MachineList{}
+			Expect(k8sClient.List(ctx, machineList, client.InNamespace(namespace))).To(Succeed())
+			Expect(machineList.Items).To(HaveLen(3))
+
+			machineToDelete := &machineList.Items[0]
+			machineToDelete.Status.NodeRef = clusterv1.MachineNodeReference{Name: "node-0"}
```

golangci-lint (2.12.2)

[error] string `node-0` has 9 occurrences, make it a constant

(goconst)